src/System.Memory/src/System/Span.cs

```diff
@@ -342,7 +342,9 @@ public bool TryCopyTo(Span<T> destination)
                     bool isOverlapped = (sizeof(IntPtr) == sizeof(int)) ? tailDiff.ToInt32() < 0 : tailDiff.ToInt64() < 0;
                     if (!isOverlapped)
                     {
-                        Unsafe.CopyBlock<T>(ref dst, ref src, (uint)length);
+                        void* pSrc = Unsafe.AsPointer<T>(ref src);
+                        void* pDst = Unsafe.AsPointer<T>(ref dst);
+                        Unsafe.CopyBlockUnaligned(pDst, pSrc, (uint)(length * Unsafe.SizeOf<T>()));
```

Comment:
CopyBlockUnaligned. It will do the same thing as CopyBlock on x86 and x64 because of these platforms do not require alignment.